### PR TITLE
Replace all logic that checks Null with the ObjectUtil utility class

### PR DESCRIPTION
**Pull request by qeesung**
Clean the code , replace all logic that checks Null with the ObjectUtil utility class in bootstrap package

Modification:
Replace all logic that checks Null with the ObjectUtil utility class

### CODE REVIEW

**Comment by normanmaurer** [comment]
@netty-bot test this please

File: transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java [hunk 1]
```diff
@@ -100,9 +99,7 @@ private B self() {
      * {@link Channel} implementation has no no-args constructor.
      */
     public B channel(Class<? extends C> channelClass) {
-        if (channelClass == null) {
-            throw new NullPointerException("channelClass");
-        }
+        ObjectUtil.checkNotNull(channelClass, "channelClass");
         return channelFactory(new ReflectiveChannelFactory<C>(channelClass));
```
Comment (normanmaurer) [inline transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java]: @qeesung: nit you can merge the above 2 lines as `checkNotNull` returns `channelClass`.

File: transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java [hunk 2]
```diff
@@ -272,9 +263,7 @@ public ChannelFuture bind(InetAddress inetHost, int inetPort) {
      */
     public ChannelFuture bind(SocketAddress localAddress) {
         validate();
-        if (localAddress == null) {
-            throw new NullPointerException("localAddress");
-        }
+        ObjectUtil.checkNotNull(localAddress, "localAddress");
         return doBind(localAddress);
```
Comment (normanmaurer) [inline transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java]: @qeesung: nit you can merge the above 2 lines

File: transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java [hunk 3]
```diff
@@ -375,9 +364,7 @@ public void run() {
      * the {@link ChannelHandler} to use for serving the requests.
      */
     public B handler(ChannelHandler handler) {
-        if (handler == null) {
-            throw new NullPointerException("handler");
-        }
+        ObjectUtil.checkNotNull(handler, "handler");
         this.handler = handler;
```
Comment (normanmaurer) [inline transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java]: nit you can merge the above two lines

File: transport/src/main/java/io/netty/bootstrap/ServerBootstrap.java [hunk 4]
```diff
@@ -130,9 +125,7 @@ public ServerBootstrap group(EventLoopGroup parentGroup, EventLoopGroup childGro
      * Set the {@link ChannelHandler} which is used to serve the request for the {@link Channel}'s.
      */
     public ServerBootstrap childHandler(ChannelHandler childHandler) {
-        if (childHandler == null) {
-            throw new NullPointerException("childHandler");
-        }
+        ObjectUtil.checkNotNull(childHandler, "childHandler");
         this.childHandler = childHandler;
```
Comment (normanmaurer) [inline transport/src/main/java/io/netty/bootstrap/ServerBootstrap.java]: nit: you can merge the above two lines

File: transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java (hunk 1 above)
Comment (qeesung) [reply transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java]: Done

File: transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java (hunk 2 above)
Comment (qeesung) [reply transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java]: Done

File: transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java (hunk 3 above)
Comment (qeesung) [reply transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java]: Done

File: transport/src/main/java/io/netty/bootstrap/ServerBootstrap.java (hunk 4 above)
Comment (qeesung) [reply transport/src/main/java/io/netty/bootstrap/ServerBootstrap.java]: Done

**Comment by normanmaurer** [comment]
@netty-bot test this please

**Pull request merged**